vscode-dotnet-runtime-library/src/Acquisition/AcquisitionInvoker.ts

```diff
@@ -33,7 +33,8 @@ export class AcquisitionInvoker extends IAcquisitionInvoker {
         const installCommand = await this.getInstallCommand(installContext.version, installContext.installDir);
         return new Promise<void>((resolve, reject) => {
             try {
-                cp.exec(winOS ? `powershell.exe -ExecutionPolicy unrestricted -File ${installCommand}` : installCommand,
+                const windowsFullCommand = `powershell.exe -ExecutionPolicy unrestricted -Command "& { [Net.ServicePointManager]::SecurityProtocol = 'Tls12, Tls13' ; & ${installCommand} }`;

```

Comment:
The following part fails if Tls1.3 is not defined (.NET Framework 4.6):
```
[Net.ServicePointManager]::SecurityProtocol = 'Tls12, Tls13'
```

I suggest we replace it with this (or equivalent):
```
    [System.Net.ServicePointManager]::SecurityProtocol = [System.Net.SecurityProtocolType]::Tls12;
    [System.Net.ServicePointManager]::SecurityProtocol += [System.Net.SecurityProtocolType]::Tls13;
```
Second line has no effect if `[System.Net.SecurityProtocolType]::Tls13` is undefined.

Reply:
The following seems to work for me (although I haven't been able to verify that it fixes the issue):

[System.Net.ServicePointManager]::SecurityProtocol=[System.Net.SecurityProtocolType]::Tls12+[System.Net.SecurityProtocolType]::Tls13

You can verify that it doesn't matter if Tls13 is not defined (as also indicated by bozturk) by running this:
`[System.Net.SecurityProtocolType]::Tls12 +[System.Net.SecurityProtocolType]::Tls13 +[System.Net.SecurityProtocolType]::ThisPropertyDoesNotExist`
... which for me gives the result "Tls12, Tls13" and does not complain about the non-existent property.

Reply:
Sounds good, I've updated this PR.